scripts/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+Dockerfile
```

Comment:
any particular reason to exclude this here? (I can imagine one, but based on that I doubt it should be excluded)